File: python/7 kyu/78.py
# https://www.codewars.com/kata/58b8c94b7df3f116eb00005b/train/python

# Passed

def reverse_letter(st):
    result = str()
    for char in st[::-1]:
        if char.isalpha():
            result += char
    return result

output = reverse_letter("ultr53o?n")
print(output)

File: nodejs/7 kyu/78.js
// https://www.codewars.com/kata/5809b62808ad92e31b000031/train/javascript

// Passed

function calculate(str) {
  numbers = str
    .replaceAll("minus", "plus-")
    .split("plus")
    .map((stringNumber) => Number(stringNumber));
  const finalSum = numbers.reduce((prev, curr) => prev + curr, 0);
  return String(finalSum);
}

const output = calculate("1plus2plus3minus4");
console.log(output);